src/Spatial.Tests/Euclidean/Line2DTests.cs

```diff
@@ -56,6 +56,20 @@ public void LineDirection(string p1s, string p2s, string exs)
             AssertGeometry.AreEqual(ex, line.Direction);
         }
 
+        [TestCase("-1,+1", "+1,+1", "0,0", 1.0)]
+        [TestCase("-1,+1", "+1,+1", "0,3", 2.0)]
+        [TestCase("-1,0", "0,+1", "0,0", 0.70710678)]
+        [TestCase("+1,-1", "+1,+1", "0,0", 1.0)]
+        [TestCase("+1,-1", "+1,+1", "3,0", 2.0)]
+        public void DistanceFromLineToPoint(string p1s, string p2s, string ps, double expectedDistance)
```

Comment:
I would add other test cases, for which the expected distance is 0, i.e. the line passes through the point.

I would also add some tests for which the order of the start/end line points is reversed, but the expected distance doesn't change.

Reply:
Thanks for your comment.

The test cases you described are meaningfull.
I'll do it, thanks.

Reply:
> I would add other test cases, for which the expected distance is 0, i.e. the line passes through the point.

Done in the commit https://github.com/f-frhs/mathnet-spatial/commit/00ce8a6e42ca608cbb126efc31b468d2d3aea84f.

Reply:
> I would also add some tests for which the order of the start/end line points is reversed, but the expected distance doesn't change.

Don in the commit https://github.com/f-frhs/mathnet-spatial/commit/ffba33569664f732f81f8c4f1452d42297fd7466